teekize/Questionner
# #163171757 Add coverage and maintainability badges

## What  does this PR do?
Adds a maintainability badge and a coverage badge on the readme file

## Description of task to be completed
add a coverage and maintainability badge on the readme file

## How to manually test this ?
1. after cloning this repo 
`https://github.com/teekize/Questionner.git`

2. cd into the project directory

3. install a virtual environment and activate the environment
`venv\Scripts\activate`

4. checkout into this branch
`git checkout ch-add- badges-163171757`

5. Open the readme.file and you will see the markdown for the badges

## Relevant  pivotal tracker stories
https://www.pivotaltracker.com/n/projects/2235429/stories/163171757
